ggml/src/ggml-cuda/cpy.cu

```diff
@@ -82,13 +83,25 @@ static __device__ void cpy_blck_f32_q8_0(const char * cxi, char * cdsti) {
 }
 
 static __device__ void cpy_blck_q8_0_f32(const char * cxi, char * cdsti) {
-    const block_q8_0 * xi = (const block_q8_0 *) cxi;
-    float * dsti = (float *) cdsti;
+    float* cdstf = (float*)(cdsti);
+
+    for (int j = 0; j < QK8_0; j+=2) {
+        float2 dq;
```

Comment:
```suggestion
        dfloat2 dq;
```

I plan to remove `dfloat2` and `GGML_CUDA_F16` in the medium term but as it is this code would fail to compile with `GGML_CUDA_F16`.